extensions/bigtable/install_hbase_bigtable.sh

```diff
@@ -78,3 +80,48 @@ echo -e "YARN_OPTS=\"\${YARN_OPTS} -Dyarn.app.mapreduce.am.command-opts=\"${BIGT
 # the hbase classpath.
 echo -e "HBASE_OPTS=\"\${HBASE_OPTS} ${BIGTABLE_BOOT_OPTS}\"" >> "${HBASE_CONF_DIR}/hbase-env.sh"
 
+# Configure Spark
+if [ -d "/home/hadoop/spark-install/" ]; then
+    SPARK_HOME="/home/hadoop/spark-install"
+    export MASTER=spark:\/\/${MASTER_HOSTNAME}:7077
+    export SPARK_LOCAL_IP=${MASTER_HOSTNAME}
```

Comment:
If you enclose the values with double quotes, you can get rid of some escaping - 
export MASTER="spark://${MASTER_HOSTNAME}:7077"